cdap-spark-core/src/main/scala/co/cask/cdap/app/runtime/spark/SparkMainWrapper.scala

```diff
@@ -214,7 +214,7 @@ object SparkMainWrapper {
           executor.awaitTermination(5L, TimeUnit.SECONDS)
           // Send the complete call
           client.completed(workflowToken)
-          LOG.info("Spark program {} completed. Program RunID: [{}]", programRunId.getProgram, programRunId.getRun)
+          LOG.info("Spark program {} completed. Program RunID: [{}]", programRunId.getProgram, programRunId.getRun:Any)
```

Comment:
Oh, it's Scala. You can do `Array(programRunId.getProgram, programRunId.getRun)`, which I think is cleaner.